native/chat/chat-thread-background.react.js

```diff
@@ -34,10 +36,31 @@ export default function BackgroundChatThreadList(
     props.navigation.setOptions({ title });
   }, [props.navigation, unreadBackgroundThreadsNumber]);
 
+  const styles = useStyles(style);
+  const emptyItem: React.Node = (
+    <Text style={styles.emptyList}>
+      {' '}
+      Background threads are just like normal threads, except they appear in
+      this tab instead of Home, and they don&apos;t contribute to your unread
+      count. {'\n'}
+      To move a thread over here, switch the Background option in its settings.
+    </Text>
+  );
+
   return (
     <ChatThreadList
       navigation={props.navigation}
       filterThreads={threadInBackgroundChatList}
+      emptyItem={emptyItem}
     />
   );
 }
+
+const style = {
```

Comment:
Let's call this `styles` for consistency with other places

Reply:
Since we can't have `const styles = useStyles(styles);`, one of them must be inconsistent ( in `native/chat/new-messages-pill.react.js` we have `const styles = useStyles(unboundStyles);` )

Reply:
Right, good catch. Let's call it `unboundStyles` then